exact_solvers/traffic_ramps.py

```diff
@@ -1,8 +1,10 @@
+import sys, os
+top_dir = os.path.abspath('..')
+if top_dir not in sys.path:
+    sys.path.append(top_dir)
```

Comment:
Why is this necessary now?  Since it adds boilerplate code to so many files and messes with the user's PYTHONPATH, it would be nice if we could avoid it...

Reply:
I guess it works ok without this now.  For some reason I was having a lot of relative import problems earlier.  I'll try removing all this and test it all out again.  Stay tuned...

Reply:
Yes it works ok and I've removed all these lines in my latest commit.